peer/brontide.go

```diff
@@ -1245,6 +1249,53 @@ func (p *Brontide) maybeSendNodeAnn(channels []*channeldb.OpenChannel) {
 	}
 }
 
+// maybeSendChannelUpdates sends our channel updates to the remote peer if we
+// have any active channels with them.
+func (p *Brontide) maybeSendChannelUpdates(channels []*channeldb.OpenChannel) {
+	defer p.wg.Done()
+
+	// If we don't have any active channels, then we can exit early.
+	if len(channels) == 0 {
+		return
+	}
+
+	for _, dbChan := range channels {
+		switch {
+		// Skip anything that isn't the default channel state.
+		case !dbChan.HasChanStatus(channeldb.ChanStatusDefault):
+			continue
+
+		// If we need to do an SCB flow, a channel update is useless.
+		case !dbChan.HasChanStatus(channeldb.ChanStatusRestored):
+			continue
+		}
+
+		// Now that we know the channel is in a good state, we'll try
+		// to fetch the update to send to the remote peer. If the
+		// channel is pending, and not a zero conf channel, we'll get
+		// an error here which we'll ignore.
+		chanUpd, err := p.cfg.FetchLastChanUpdate(dbChan.ShortChannelID)
```

Comment:
For zero-conf channels, before 6 confs this will be the very first alias scid. After six confs, the on-chain scid will be stored in `OpenChannel.confirmedScid`. Things like this make me regret doing zero-conf scid's the way I did, we should probably clean it up. So to grab the correct scid for zero-conf channels, you call `ZeroConfConfirmed` and if it's true, use the scid returned from `ZeroConfRealScid`. If it's false, then you do what you did here.

Reply:
> Things like this make me regret doing zero-conf scid's the way I did, we should probably clean it up.

Hmm....how would we go about doing that? Can we make a tracking issue for this? Seems like instead we should make the scid attribute internal, then force resolution through another method that's aware of which one it should return when. 

Will update the logic to call `ZeroConfConfirmed`.